packages/devtools/lib/src/memory/memory.dart

```diff
@@ -362,38 +682,357 @@ class MemoryScreen extends Screen with SetStateMixin {
       );
     }
 
-    table.onSelect.listen((InstanceSummary row) async {
-      ga.select(ga.memory, ga.inspectInstance);
+    table.onCellHover.listen(hoverInstanceAllocations);
+    table.onSelect.listen(select);
 
-      // User selected a new instance from the list of class instances so the
-      // instance view which would be the third child needs to be removed.
-      _removeInstanceView();
+    return table;
+  }
 
-      Instance instance;
-      try {
-        instance = await memoryController.getObject(row.objectRef);
-      } catch (e) {
-        instance = null; // Signal a problem
-      } finally {
-        tableContainer.add(_createInstanceView(
-          instance != null
-              ? row.objectRef
-              : 'Unable to fetch instance ${row.objectRef}',
-          row.className,
-        ));
-
-        tableContainer.element.scrollTo(<String, dynamic>{
-          'left': tableContainer.element.scrollWidth,
-          'top': 0,
-          'behavior': 'smooth',
-        });
+  void select(InstanceSummary row) async {
+    ga.select(ga.memory, ga.inspectInstance);
+
+    // User selected a new instance from the list of class instances so the
+    // instance view which would be the third child needs to be removed.
+    _removeInstanceView();
+
+    Instance instance;
+    try {
+      instance = await memoryController.getObject(row.objectRef);
+
+      final dynamic theObject = await memoryController.getObject(row.objectRef);
+      if (theObject is Instance) {
+        instance = theObject;
+      } else if (theObject is Sentinel) {
+        instance = null;
+        // TODO(terry): Tracking Sentinel's to be removed.
+        framework.toast('Sentinel ${row.objectRef}', title: 'Warning');
+      }
+    } catch (e) {
+      instance = null; // Signal a problem
```

Comment:
should we actually display an error or at least log it to analytics so we can understand if users are hitting this.

Reply:
logging to analytics.